libp2p/protocols/pubsub/rpc/messages.nim

```diff
@@ -113,7 +113,7 @@ func shortLog*(msg: Message): auto =
     fromPeer: msg.fromPeer.shortLog,
     data: msg.data.shortLog,
     seqno: msg.seqno.shortLog,
-    topicIds: $msg.topicIds,
+    topicId: $msg.topicId,
```

Comment:
I'm nitpicking, but `$` isn't necessary anymore, topicId is already a string.

Reply:
Do nitpick, that's what we're here for :)